src/lightning_app/core/flow.py

```diff
@@ -608,3 +603,36 @@ def experimental_iterate(self, iterable: Iterable, run_once: bool = True, user_k
             yield value
 
         self._calls[call_hash].update({"has_finished": True})
+
+    def configure_commands(self):
+        """Configure the commands of this LightningFlow.
+
+        Returns a list of dictionaries mapping a command name to a flow method.
```

Comment:
Why a list? The order has no influence? Can we just keep it to a dict?

Reply:
There are several commands. This is a list to keep the same overall API as with layouts. I agree order doesn't matter, but you don't need to learn 2 ways of doing something.